_data/identity.yml

```diff
@@ -97,6 +97,17 @@ websites:
       - hardware
     doc: https://help.okta.com/en/prod/Content/Topics/Security/MFA.htm
 
+  - name: OneLogin
+    url: https://www.onelogin.com/
+    img: onelogin.png
+    tfa:
+      - proprietary
+      - sms
+      - totp
+      - hardware
+      - u2f
```

Comment:
Yubikeys are mentioned but the phrasing makes it sound like it's their normal OTP solution. Other than that no U2F solution is mentioned.
```suggestion
```